tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -143,6 +143,15 @@ def tap_icon(self):
             Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == expected_manifest_url)
             self.apps.switch_to_displayed_app()
 
+        def long_tap(self):    
```

Comment:
This function has duplicated some of its logic from `move_to()`. We can reduce that, this way:
```py
def activate_edit_mode():
    self._start_long_press_action().\
        release().\
        perform()    

def move_to(self, html_element):
    self._start_long_press_action().\
        move(html_element).\
        wait(1).\
        release().\
        wait(1).\
        perform()

def _start_long_press_action(self):
    self.marionette.execute_script('arguments[0].scrollIntoView(false);', [self.root_element])

    return Actions(self.marionette).\
        press(self.root_element).\
        wait(3)
```

This will also enable us to show more the user's intent behind `long_tap()`